ff-filter-cloud/styles.css

```diff
@@ -0,0 +1,222 @@
+    background-color: #f6f6f6;
+    border-bottom: 1px solid #ededed;
+}
+
+div[slot="selected"] span.filterName:hover {
+    text-decoration: underline;
+}
+
+div[slot="unselected"] span.filterName:hover {
+    text-decoration: underline;
+}
+
+.filterCheckboxUnselected {
+    background-image: url("../../resources/images/sprite-sf85bcdc069.png");
+    background-repeat: no-repeat;
+    background-position: 0 -68px;
+    height: 12px;
+    width: 12px;
+    display: inline-block;
+    margin-right: 6px;
+    position: relative;
+    top: 1px;
+}
+
+.filterCheckboxSelected {
+    background-image: url("../../resources/images/sprite-sf85bcdc069.png");
+    background-repeat: no-repeat;
+    background-position: -13px -41px;
+    height: 12px;
+    width: 12px;
+    content: '';
+    display: block;
+    left: 0;
+    position: absolute;
+    top: 0;
+}
+
+[data-container^="show"] span.text {
+    display: inline-block;
+    color: #5c5c5c;
+    padding: 8px 0 8px 15px;
+}
+
+[data-container="showMore"] {
+    transition: 400ms;
+}
+
+[data-container="hiddenLinks"] {
+    transition: 400ms;
+}
+
+[data-container="removeFilter"] {
+    font-size: 80%;
+    color: #5c5c5c;
+    padding: 7px 0 0 15px;
+    cursor: pointer;
+}
+
+.groupCaption {
+    padding: 0 0 0 12px;
+    color: #666;
+    position: relative;
+    background-color: #ebebeb;
+    height: 25px !important;
+    border-bottom-radius: 7px;
+    line-height: 25px;
+    vertical-align: middle;
+}
+
+.filterArrowRight {
+    background-position: 0 -22px;
+    height: 7px;
+    width: 4px;
+    display: inline-block;
+    bottom: 1px;
+    position: relative;
+    background-image: url("../../resources/images/sprite-sf85bcdc069.png");
+    background-repeat: no-repeat;
+}
+
+.filterArrowDown {
+    background-position: -17px -22px;
+    height: 9px;
+    width: 15px;
+    background-image: url("../../resources/images/sprite-sf85bcdc069.png");
+    background-repeat: no-repeat;
+    position: absolute;
+    right: 5px;
+    top: 8px;
+    display: block;
+}
+
+.clusterLevel1 {
+    margin-left: 10px;
+}
+
+.clusterLevel2 {
+    margin-left: 20px;
+}
+
+.cursor {
+    cursor: pointer;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+}
+
+main {
+    display: flex;
+    flex-direction: row;
+}
+
+nav {
+    width: 210px;
+}
+
+section {
+    padding: 0 15px;
+    display: flex;
+    flex-direction: column;
+}
+
+.bctContainer {
+    margin: 10px 0 10px 10px;
+}
+
+ff-breadcrumb-trail {
+    margin-left: 10px;
+}
+.fc-header {
+    font-weight: bold;
+}
+.fc-btn-deselect {
+    cursor: pointer;
+    padding: 5px;
+}
+
+ff-filter-cloud {
+    width: 100%;
+    display:flex;
+    flex-direction: column;
+}
+
+.fc-container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+}
+
+.fc-filter-item {
+    color: #5C5C5C;
+    background-color: #C7E4EC;
+    width: fit-content;
+    border-radius: 10px;
```

Comment:
[optional] For consistency `5px` might fit better in the whole picture.

Reply:
I will leave as it is now. With 5px radius they look like buttons.